tests/unit/s2n_ktls_io_test.c

```diff
@@ -1185,5 +1203,107 @@ int main(int argc, char **argv)
         };
     };
 
+    /* Test: key encryption limit tracked */
+    {
+        uint8_t large_test_data[S2N_TLS_MAXIMUM_FRAGMENT_LENGTH * 10] = { 0 };
+        const size_t large_test_data_records = sizeof(large_test_data) / S2N_TLS_MAXIMUM_FRAGMENT_LENGTH;
```

Comment:
```suggestion
        const size_t large_test_data_records = 10;
```
Feels like you shouldn't need this calculation

Reply:
Sure, but then I have to write 10 twice lol. And i can't define the size of large_test_data based on large_test_data_records, because array sizes have to be static.

But if you don't mind the duplicate magic 10, I can write it twice.